Bearded.Utilities/Noise/NoiseMap.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+namespace Bearded.Utilities.Noise
+{
+    public static class NoiseMap
+    {
+        public static INoiseMap FromArray(double[,] array, IInterpolationMethod2 interpolation) =>
```

Comment:
This really makes me question the name even more, sounds more like this is a ContinuousMap (or maybe not even a Map?), not a NoiseMap.